phpunit.xml.dist

```diff
@@ -34,6 +34,7 @@ It's auto-generated by sonata-project/dev-kit package.
 
     <php>
         <ini name="precision" value="8"/>
+        <env name="SYMFONY_PHPUNIT_REMOVE" value="symfony/yaml" />
```

Comment:
@OskarStark I followed the doc https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script

Without this, I can't run the tests, because `./vendor/bin/simple-phpunit` does not include `prophecy`.